conf/github-contributions.nginx.conf

```diff
@@ -26,4 +35,14 @@ server {
         alias /github-archive/timeline/;
         autoindex on;
     }
+
+    location ~ /.well-known {
+        allow all;
+    }
```

Comment:
Oh and also, the `.well-known` thing is where the letsencrypt client shoves some file so that their verifying server can verify that we still have the domain.